.github/workflows/unittest.yaml

```diff
@@ -0,0 +1,41 @@
+name: unittest
+
+on:
+  pull_request:
+    branches: [ main ]
+  paths-ignore:
```

Comment:
I think this needs to be indented to the same level as `branches`.

Reply:
ack